### PR TITLE
[Merged by Bors] - style(test/polyrith): adapt to mathlib4 style

### PR DESCRIPTION
**Pull request by grunweg**
Adapt indentation to mathlib4 conventions. Found when clearing out old branches.

---------

The first 250 lines are not handled; I think that developping an auto-formatter for Lean is a better use of time than fixing these by hand.

The diff in the last lines is slightly confusing; there are only indentation changes.

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by rosborn — APPROVED** [review]
LGTM

File: test/polyrith.lean [hunk 1]
```diff
@@ -274,8 +274,8 @@ A full test suite is provided at the bottom of the file.
 
 -- constant term : ∀ a b : ℚ, a + b = 0
 
--- example (a b c d : ℚ) (h : a + b = 0) (h2: b + c = 0): a + b + c + d = 0 :=
--- by test_polyrith only [term c d, h]
+-- example (a b c d : ℚ) (h : a + b = 0) (h2: b + c = 0): a + b + c + d = 0 := by
```
[the remark this answers is not in the document]
Comment (grunweg) [reply test/polyrith.lean]: I hope so too!

**Comment by grunweg** [comment]
Thank you for the keen eye; I fixed all those issues.

**Comment by urkud** [comment]
Thanks! :tada:
bors merge

**Pull request closed without merging**